manuscript/recipes/jellyfin.md

```diff
@@ -0,0 +1,98 @@
+# Jellyfin
+
+[Jellyfin](https://jellyfin.org/) is best described as "_like [Emby](/recipes/emby) but really FOSS.
+
+![Jellyfin Screenshot](../images/jellyfin.png)
+
+If it looks very similar as Emby, is because it started as a fork of it, but it has evolve since them. For a complete explanation of the why, look [here](https://jellyfin.org/docs/general/about.html).
+
+## Ingredients
+
+1. [Docker swarm cluster](/ha-docker-swarm/design/) with [persistent shared storage](/ha-docker-swarm/shared-storage-ceph.md)
+2. [Traefik](/ha-docker-swarm/traefik) configured per design
+3. DNS entry for the hostname you intend to use, pointed to your [keepalived](ha-docker-swarm/keepalived/) IP
+
+## Preparation
+
+### Setup data locations
+
+We'll need a location to store Jellyfin's library data, config files, logs and temporary transcoding space, so create /var/data/jellyfin, and make sure it's owned by the user and group who also own your media data.
+
+```
+mkdir /var/data/jellyfin
+```
+
+Also if we want to avoid the cache to be part of the backup, we should create a location to map it on the runtime folder. It also has to be owned by the user and group who also own your media data.
+
+```
+mkdir /var/data/runtime/jellyfin
+```
+
+### Prepare environment
+
+Create jellyfin.env, and populate with PUID/GUID for the user who owns the /var/data/jellyfin directory (_above_) and your actual media content (_in this example, the media content is at **/srv/data**_)
+
+```
+PUID=
+GUID=
+```
+
+### Setup Docker Swarm
+
+Create a docker swarm config file in docker-compose syntax (v3), something like this:
+
+!!! tip
+        I share (_with my [sponsors](https://github.com/sponsors/funkypenguin)_) a private "_premix_" git repository, which includes necessary docker-compose and env files for all published recipes. This means that sponsors can launch any recipe with just a ```git pull``` and a ```docker stack deploy``` 👍
+
+
+```
+version: "3.0"
+
+services:
+  jellyfin:
+    image: jellyfin/jellyfin
+    env_file: /var/data/config/jellyfin/jellyfin.env
+    volumes:
+      - /etc/localtime:/etc/localtime:ro
+      - /var/data/jellyfin:/config
+      - /var/data/runtime/jellyfin:/cache
+      - /var/data/jellyfin/jellyfin:/config
+      - /srv/data/:/data
+    deploy:
+      labels:
+        - traefik.frontend.rule=Host:jellyfin.example.com
+        - traefik.docker.network=traefik_public
+        - traefik.port=8096
+    networks:
+        - traefik_public
+        - internal
+    ports:
+      - 8096:8096
+
+networks:
+  traefik_public:
+    external: true
+  internal:
+    driver: overlay
+    ipam:
+      config:
+        - subnet: 172.16.67.0/24
```

Comment:
Add the subnet to https://github.com/geek-cookbook/geek-cookbook/blob/master/manuscript/reference/networks.md (and use the next available one)